doc/transform.ipynb

```diff
@@ -2335,8 +2335,14 @@
    "cell_type": "markdown",
```

Comment:
<p>I think "regular depth grid" might be better? Also, by regular do you mean the distances are regularly spaced?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Q1: Yes, "regular depth grid" would be better.</p><p> </p><p>Q2: I did mean a regularly spaced vertical grid, but maybe "regular z-level vertical grid" would be a better description?</p>

Reply:
I like "regularly spaced vertical grid" or "uniformly spaced..."

Reply:
<p>+1 for uniformly spaced</p>

Reply:
<p>How's this for the final version: "Interpolating to uniformly spaced vertical grid"</p><p> </p><p> </p>